### PR TITLE
📝 Add info on egress costs for Cloud storage connectors

### PR DESCRIPTION
**Pull request by alex-gron**
## What

- Add info blurb to GCS, S3, and Azure Blob Storage source and destination docs
- This info links to pricing documentation for each Cloud provider
- This is indented to bring more transparency to users of these connectors

### CODE REVIEW

File: docs/integrations/destinations/s3.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This page guides you through the process of setting up the S3 destination connector.
 
+::::info
+Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Amazon S3 pricing guide](https://aws.amazon.com/s3/pricing/).
+::::
```
Comment (bnchrch) [inline docs/integrations/destinations/s3.md:7]:
```suggestion
:::info
Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Amazon S3 pricing guide](https://aws.amazon.com/s3/pricing/).
:::
```

File: docs/integrations/destinations/gcs.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ This destination writes data to GCS bucket.
 
 The Airbyte GCS destination allows you to sync data to cloud storage buckets. Each stream is written to its own directory under the bucket.
 
+::::info
+Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Google Cloud Storage pricing guide](https://cloud.google.com/storage/pricing).
+::::
```
Comment (bnchrch) [inline docs/integrations/destinations/gcs.md:11]:
```suggestion
:::info
Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Google Cloud Storage pricing guide](https://cloud.google.com/storage/pricing).
:::
```

File: docs/integrations/destinations/azure-blob-storage.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ This destination writes data to Azure Blob Storage.
 
 The Airbyte Azure Blob Storage destination allows you to sync data to Azure Blob Storage. Each stream is written to its own blob under the container.
 
+::::info
+Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Azure Blob Storage pricing guide](https://azure.microsoft.com/en-us/pricing/details/storage/blobs/).
+::::
```
Comment (bnchrch) [inline docs/integrations/destinations/azure-blob-storage.md:11]:
```suggestion
:::info
Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Azure Blob Storage pricing guide](https://azure.microsoft.com/en-us/pricing/details/storage/blobs/).
:::
```

File: docs/integrations/sources/gcs.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 This page guides you through the process of setting up the GCS source connector. This connector supports loading multiple CSV files (non compressed) from a GCS directory. The conntector will check for all files ending in `.csv`, even nested files.
 
+::::info
+Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Google Cloud Storage pricing guide](https://cloud.google.com/storage/pricing).
+::::
```
Comment (bnchrch) [inline docs/integrations/sources/gcs.md:7]:
```suggestion
:::info
Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Google Cloud Storage pricing guide](https://cloud.google.com/storage/pricing).
:::
```

File: docs/integrations/sources/s3.md [hunk 5]
```diff
@@ -2,6 +2,10 @@
 
 This page contains the setup guide and reference information for the Amazon S3 source connector.
 
+::::info
+Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Amazon S3 pricing guide](https://aws.amazon.com/s3/pricing/).
+::::
```
Comment (bnchrch) [inline docs/integrations/sources/s3.md:7]:
```suggestion
:::info
Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Amazon S3 pricing guide](https://aws.amazon.com/s3/pricing/).
:::
```

**Review by bnchrch — APPROVED** [review]
Well done! 

The only thing I noticed was the extra colons.

Outside of that this looks good to me. Preapproved.

File: docs/integrations/sources/azure-blob-storage.md [hunk 6]
```diff
@@ -2,6 +2,9 @@
 
 This page contains the setup guide and reference information for the Azure Blob Storage source connector.
 
+::::info
+Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Azure Blob Storage pricing guide](https://azure.microsoft.com/en-us/pricing/details/storage/blobs/).
+::::
```
Comment (alex-gron) [inline docs/integrations/sources/azure-blob-storage.md:7]:
```suggestion
:::info
Cloud storage may incur egress costs. Egress refers to data that is transferred out of the cloud storage system, such as when you download files or access them from a different location. For more information, see the [Azure Blob Storage pricing guide](https://azure.microsoft.com/en-us/pricing/details/storage/blobs/).
:::
```

**Review by evantahler — APPROVED** [review]

**Pull request merged**